circuits/core/events.py

```diff
@@ -18,6 +18,10 @@ class Event(object):
     alert_done = False
     waitingHandlers = 0
 
+    # Behaviour for handling errors:
+    # One of "abort", "igore" (default)
```

Comment:
If "ignore" is the default, wouldn't it make sense to set `on_error` to that instead of None?